tests/external/test_server_connection.py

```
@@ -0,0 +1,54 @@
+            zotero_library_type=os.getenv("TEST_ZOTERO_LIBRARY_TYPE", "user"),
+        )
+
+        client = ZoteroClient(real_settings)
+
+        assert client.mode == "web"
+        assert isinstance(client._client, zotero.Zotero)
+
+        try:
+            items = client._client.items(limit=1)
+            assert isinstance(items, list)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)